Cargo.toml

```diff
@@ -7,13 +7,28 @@ authors = ["Yi Zhou <me@y1zhou.com>"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[lib]
+name = "arpeggia"
+crate-type = ["cdylib", "rlib"]
+
+[[bin]]
+name = "arpeggia"
+path = "src/main.rs"
+required-features = []
+
 [dependencies]
 clap = { version = "4.5.29", features = ["derive"] }
 nalgebra = "0.33.2"
 pdbtbx = { version = "0.12.0", features = ["rayon", "rstar"] }
-polars = { version = "0.50.0", features = ["lazy", "parquet", "json"] }
+polars = { version = "0.48.0", features = ["lazy", "parquet", "json"] }
```

Comment:
The latest rust polars version is 0.52.0. What disables us from using newer versions?